Gruntfile.js

```diff
@@ -12,6 +12,12 @@ module.exports = function (grunt) {
                 cwd: 'src',
                 src: 'styles/dgrid.css',
                 dest: '<%= distDirectory %>'
+            },
+            examples: {
```

Comment:
> Should this be called staticExampleFiles like in widgets?

I think we're fine inlining static values that are specific to the dgrid project, as is already being done just above for `devStyles.src` and `distStyles.src`.